docs/observability/app-metrics.md

```diff
@@ -1,60 +1,71 @@
+# Application Metrics
+
 This guide describes the Prometheus metrics endpoint exposed by Mermin and provides a comprehensive breakdown of all available metrics, their types, and descriptions.
 
 ## Metrics Endpoint
 
-Mermin exposes Prometheus metrics in the standard Prometheus text format at the `/metrics` HTTP endpoint. The metrics server is enabled by default and listens on port `10250` (configurable via the `metrics.port` configuration option).
+Mermin exposes Prometheus metrics in the standard Prometheus text format at multiple HTTP endpoints:
+
+- `/metrics` - All metrics (standard + debug if enabled)
+- `/metrics/standard` - Standard metrics only (no high-cardinality labels)
+- `/metrics/debug` - Debug metrics only (returns 404 if disabled)
+- `/metrics:summary` - JSON summary of all available metrics with metadata (name, type, description, labels, category) (AIP-136 custom method)
```

Comment:
We don't need to call out that this is an AIP design in the user docs. No one cares about that.

```suggestion
- `/metrics:summary` - JSON summary of all available metrics with metadata (name, type, description, labels, category)
```